package.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
- [ ] Your app appears not properly setup. _Webpack_  web server is missing, and essential scripts to run your app is also missing in the `package.json` file. Please ensure to add _Webpack_ and add the necessary configurations so as to complete the setup process. Your `package.json` file should include similar configurations like below:

```json
  "scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack --config webpack.config.js --watch"
  }
```